scripts/_build_pr80_ledger.py

```
@@ -0,0 +1,306 @@
+
+def require_gh() -> None:
+    if shutil.which("gh") is None:
+        raise SystemExit("error: gh CLI not found on PATH")
+
+
+def paginate(url_base: str) -> list:
+    out: list = []
+    page = 1
+    while True:
+        url = f"{url_base}?per_page=100&page={page}"
+        r = subprocess.run(  # nosemgrep: python.lang.security.audit.dangerous-subprocess-use-audit
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)